scripts/changelog.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I'd be in favor of rewriting this in JS/Node. It would be more verbose but more portable and readable...